### DISCUSSION TITLE
gitlab ci failed with "python: can't open file '/app/manage.py'"

### DISCUSSION THREAD

**Discussion by willhoh** [cookiecutter/cookiecutter-django#4291] [Q&A]
Hi,
nice template and a super start for a project. 

Everything works right away and you can follow everything very well. Unfortunately, there is only one small thing, which is probably due to my system, which does not work.
In my gitlab-ci the test stage "pytest" hangs in the gitlab-ci and throws the error: 

```shell
python: can't open file '/app/manage.py': [Errno 2] No such file or directory
2
Cleaning up project directory and file based variables
ERROR: Job failed: exit code 2
```

When I run the gitlab-ci steps locally by hand everything works without problems. 

I'm not really a pro at gitlab-ci and can't quite figure out the error here. Would you please be so kind and help me with this?

**Comment by browniebroke** [comment]
That sounds like a bug... Are you able to share a project that was generated by the template on Gitlab, showing the problem? I'm not too familiar with Gitlab but having a reproduction would be helpful, we might see something else in the logs/project layout.

Did you find the problem since then?

**Reply by willhoh** [reply]
Hi, 
[https://github.com/willhoh/cookiecutter-django-test](https://github.com/willhoh/cookiecutter-django-test)
is a project I've started for testing. Even this initial comit returns the same error.

**Reply by browniebroke** [reply]
That would have been easier if you had created the repo on gitlab and linked directly to it, but I forked it and [imported it in my Gitlab account](https://gitlab.com/browniebroke/cookiecutter-django-gitlabci-docker-test). I kicked off a build and [it passed](https://gitlab.com/browniebroke/cookiecutter-django-gitlabci-docker-test/-/pipelines/858074278).

Not sure what's the problem, I cannot reproduce it.
